rrtmgp/mo_gas_optics_rrtmgp.F90

```diff
@@ -774,6 +784,20 @@ function source(this,                               &
     call reorder123x321(lay_source_t, sources%lay_source)
     call reorder123x321(lev_source_inc_t, sources%lev_source_inc)
     call reorder123x321(lev_source_dec_t, sources%lev_source_dec)
+    if (present(sourcesJac)) then
+      !$acc enter data copyin(sourcesJac)
+      !$acc enter data create(sourcesJac%sfc_source)
+      !$acc parallel loop collapse(2)
```

Comment:
Before doing this loop we should check that sourcesJac has the same spectral discretization and number of column as the optical properties

Reply:
I placed the check into function gas_optics_int where  check on sources was performed